src/cgtop/cgtop.c

```diff
@@ -864,6 +873,11 @@ static int get_cgroup_root(char **ret) {
         const char *m;
         int r;
 
+        if (arg_root) {
+                *ret = arg_root;
```

Comment:
You should really strdup() here, so that the caller doesn't free the static variable, and ownership of the returned string is clear.